scrapy/czlscrape/settings.py

```diff
@@ -89,3 +89,7 @@
 #HTTPCACHE_DIR = 'httpcache'
 #HTTPCACHE_IGNORE_HTTP_CODES = []
 #HTTPCACHE_STORAGE = 'scrapy.extensions.httpcache.FilesystemCacheStorage'
+
+LOG_LEVEL = 'WARN'
+LOG_FORMAT = '%(asctime)s srv="czl-scrape" [%(thread)d] %(levelname)s %(name)s: %(message)s'
```

Comment:
Observations:
- i think `%(process)d` would be more useful than thread. @margelatu ?
- maybe use `%(funcName)s` instead of name?
- aaand, management wanted the "srv" part to include the scraper "id". it's true that it's redundant with name, but...
  i'm not sure how we'd do this without passing `extra` to logger.*(). maybe use a custom formatter which extracts it from {module}?